adm_program/languages/en.xml

```diff
@@ -1403,4 +1404,5 @@
   <string name="SYS_YOUR_LISTS">Own lists</string>
   <string name="SYS_YOUR_NAME">Your name</string>
   <string name="SYS_Z_TO_A">Z to A</string>
+  <string name="MEM_POSSIBLE_FIELDNAMES">Column names for autodetection: #VAR1#, #VAR2#</string>
```

Comment:
could you change the position in file so this string will be by the other MEM strings.